Automattic/jetpack
# Config Package: Allow for enabling post_list package

Fixes https://github.com/Automattic/jetpack/issues/21102
Part of #20941

#### Changes proposed in this Pull Request:

* Allows for enabling the post_list package following these steps: https://github.com/Automattic/jetpack/tree/master/projects/packages/config
* The post_list package was added here: https://github.com/Automattic/jetpack/pull/20720

#### Jetpack product discussion



N/A

#### Does this pull request change what data or activity we track or use?



N/A

#### Testing instructions:






* Follow[ the "Usage" instructions](https://github.com/Automattic/jetpack/tree/master/projects/packages/config#usage):
    * Require both the config and post_list packages:
        `cd projects/plugins/jetpack && composer require automattic/jetpack-post-list`
        `composer require automattic/jetpack-config` 
    * Install the monorepo:
        `pnpm install`
        `jetpack install -r && jetpack install --all`
    * Initialize the configuration package with the post_list slug:
        Create a file named 0-functions.php in the tools/docker/mu-plugins/ directory and add:
```
use Automattic/Jetpack/Config;

// Configuring Jetpack as early as plugins_loaded priority 1
// to make sure every action handler gets properly set.
add_action( 'plugins_loaded', 'configure_jetpack', 1 );

function configure_jetpack() {
    $config = new Config();
    $config->ensure( 'post_list' );
}
```

* Build and start the docker env:
        `jetpack build packages/post-list`
        `jetpack docker up`
* Go to http://localhost/wp-admin/edit.php (the post list) and check that the post thumbnails appear.
